content/applications/finance/sign.rst

```diff
@@ -1,5 +1,3 @@
-:show-content:
-
 ====
 Sign
 ====
```

Comment:
Proposal of improvement (open debate):
It might be interesting to specify in the beginning of the documentation of enterprise features that they are restricted to enterprise users.
It's already done in the developer doc to specify views restricted to enterprise (https://www.odoo.com/documentation/15.0/developer/reference/backend/views.html?highlight=views#cohort), but I believe it might make sense in the user doc as well.

cc: @jcs-odoo 

If it's ok for you, you can integrate the tag with the following rst code:
```rst

.. raw:: html

   <span class="badge" style="background-color:#AD5E99">Enterprise feature</span>
```

Reply:
This is indeed a good suggestion, but the user doc only focuses on Enterprise for now.
I'm worried that if we add this here for this doc only, it may mislead readers when they read other pages.

I'll keep this in mind for the future, but I wouldn't haste it in this PR right now.

Thanks